src/index.js

```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import 'patternfly/dist/css/rcue.css';
-import 'patternfly/dist/css/rcue-additions.css';
 import './styles/.css/index.css';
```

Comment:
I needed to change this to './styles/index.scss' to get the failing test to pass

Reply:
If I changed this, then the site wouldn't render. I ended up changing the path in `index.scss`.

Reply:
Oh, you wanted the site to render? :)